pyproject.toml

```diff
@@ -41,6 +41,9 @@ sphinx-toolbox = "^3.1.2"
 requires = ["poetry-core>=1.4.2", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
 build-backend = "poetry_dynamic_versioning.backend"
 
+[tool.poetry-dynamic-versioning]
+enable=true
```

Comment:
missed to enable poetry-dynamic-versioning
now you can test on local by running below command with local git tag of `vN.N.N`
```
poetry build
```

Reply:
@AlexandrosKyriakakis confirmed on local that the __version__ is replaced by the poetry-dynamic-versioning

```bash
% git clone git@github.com:hiohiohio/alpaca-py.git
% git switch fix-publish-pypi
% git tag v0.0.2      
% poetry build
Building alpaca-py (0.0.2)
  - Building sdist
  - Built alpaca_py-0.0.2.tar.gz
  - Building wheel
  - Built alpaca_py-0.0.2-py3-none-any.whl
% cd dist                  
% unzip alpaca_py-0.0.2-py3-none-any.whl
Archive:  alpaca_py-0.0.2-py3-none-any.whl
  inflating: alpaca/__init__.py      
  inflating: alpaca/broker/__init__.py  
  inflating: alpaca/broker/client.py  
  inflating: alpaca/broker/enums.py  
  inflating: alpaca/broker/models/__init__.py  
  inflating: alpaca/broker/models/accounts.py  
  inflating: alpaca/broker/models/cip.py  
  inflating: alpaca/broker/models/documents.py  
  inflating: alpaca/broker/models/funding.py  
  inflating: alpaca/broker/models/journals.py  
  inflating: alpaca/broker/models/trading.py  
  inflating: alpaca/broker/requests.py  
  inflating: alpaca/common/__init__.py  
  inflating: alpaca/common/constants.py  
  inflating: alpaca/common/enums.py  
  inflating: alpaca/common/exceptions.py  
  inflating: alpaca/common/models.py  
  inflating: alpaca/common/requests.py  
  inflating: alpaca/common/rest.py   
  inflating: alpaca/common/types.py  
  inflating: alpaca/common/utils.py  
  inflating: alpaca/common/websocket.py  
  inflating: alpaca/data/__init__.py  
  inflating: alpaca/data/enums.py    
  inflating: alpaca/data/historical/__init__.py  
  inflating: alpaca/data/historical/crypto.py  
  inflating: alpaca/data/historical/news.py  
  inflating: alpaca/data/historical/option.py  
  inflating: alpaca/data/historical/screener.py  
  inflating: alpaca/data/historical/stock.py  
  inflating: alpaca/data/historical/utils.py  
  inflating: alpaca/data/live/__init__.py  
  inflating: alpaca/data/live/crypto.py  
  inflating: alpaca/data/live/stock.py  
  inflating: alpaca/data/mappings.py  
  inflating: alpaca/data/models/__init__.py  
  inflating: alpaca/data/models/bars.py  
  inflating: alpaca/data/models/base.py  
  inflating: alpaca/data/models/news.py  
  inflating: alpaca/data/models/orderbooks.py  
  inflating: alpaca/data/models/quotes.py  
  inflating: alpaca/data/models/screener.py  
  inflating: alpaca/data/models/snapshots.py  
  inflating: alpaca/data/models/trades.py  
  inflating: alpaca/data/requests.py  
  inflating: alpaca/data/timeframe.py  
  inflating: alpaca/py.typed         
  inflating: alpaca/trading/__init__.py  
  inflating: alpaca/trading/client.py  
  inflating: alpaca/trading/enums.py  
  inflating: alpaca/trading/models.py  
  inflating: alpaca/trading/requests.py  
  inflating: alpaca/trading/stream.py  
  inflating: alpaca_py-0.0.2.dist-info/LICENSE  
  inflating: alpaca_py-0.0.2.dist-info/METADATA  
  inflating: alpaca_py-0.0.2.dist-info/WHEEL  
  inflating: alpaca_py-0.0.2.dist-info/RECORD  
% cat alpaca/__init__.py 
# placeholder for poetry-dynamic-versioning
__version__ = "0.0.2"
```